README.md

```diff
@@ -59,6 +59,8 @@ plugins:
         embed_format: '{img_open}{img_src}{img_close}'
         # Glob pattern for matching source files
         sources: '*.drawio'
+        # Link draw.io colours with theme CSS
+        link_css: false
```

Comment:
Could we rename this option and provide some more inline docs here? I realise it's documented below, but since users tend to copy and paste this snippet into their configuration it may help sign post.

```suggestion
        # Replace diagrams' default colors with the theme's
        # NOTE: requires support from theme or custom CSS
        use_theme_colors: false
```

As a Brit it pains me to drop the `u`, but I think it's probably the least bad spelling...